src/send/SendEnterAmount.tsx

```diff
@@ -315,14 +315,18 @@ function SendEnterAmount({ route }: Props) {
               />
               <Touchable
                 borderRadius={TOKEN_SELECTOR_BORDER_RADIUS}
-                onPress={onTokenPickerSelect}
+                onPress={() => {
+                  if (!forceTokenId) {
+                    onTokenPickerSelect()
+                  }
```

Comment:
Instead of this, we should just not render this component as a touchable. Otherwise the component is still tappable.